src/BinSkim.Rules.FunctionalTests/ILDiagnosticsAnalyzerTests.cs

```diff
@@ -78,6 +78,7 @@ public void RunSymbolNameAndRoslynCallbackReportingsAnalyzers()
                 "Symbol encountered in MSIL 'global::<Module>'",
                 "Symbol encountered in MSIL 'global::Microsoft.CodeAnalysis.BinaryParsers.BinaryParsersResources'",
                 "Symbol encountered in MSIL 'global::Microsoft.CodeAnalysis.BinaryParsers.Packer'",
+                "Symbol encountered in MSIL 'global::Microsoft.CodeAnalysis.BinaryParsers.PlatformSpecificHelpers'",
```

Comment:
this test file will be deleted soon, you might just want to delete it in your branch so that it doesn't continue to produce costs for you.

Reply:
I've added an ignore tag on my other branch.  :)